kpi-tracker/resources/schema.sql

```diff
@@ -1,3 +1,18 @@
+-- Floors a (bigint) timestamp in seconds into date in seconds, e.g. 15/02/2022:14:00:00 -> 15/02/2022:00:00:00
+CREATE OR REPLACE FUNCTION date_seconds(t bigint)
+  RETURNS bigint
+  LANGUAGE plpgsql
+AS $$
+DECLARE
+  date bigint;
+BEGIN
+  SELECT floor((t)/86400)*86400::bigint
```

Comment:
So if we are being pedants, is this correct?

there are these things called leap seconds which might mess with this. So not all days have 86400 seconds.

Reply:
This is the code grafana generates from their macros when you define that you want time grouped by days, so that's where I got it. I guess you could also just define this as grouping per 24 hours (instead of date)?